### ISSUE TITLE
Test_order_transparent failed with VTK 8.2.0

### ISSUE THREAD

**Issue by skoudoro** [fury-gl/fury#84]
## Description

It seems that one of our tests is failing because of VTK 8.2.0. We need to check the color rendering with VTK 8.2 and then, we may have to update this test.

Below, the error that you can find [here](https://travis-ci.org/fury-gl/fury/jobs/556006338#L3339)
```
        # therefore the green component must have a higher value (in RGB terms)
>       npt.assert_equal(arr[150, 150][1] > arr[150, 150][0], True)
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: False
E        DESIRED: True
```

**Comment by Garyfallidis** [comment]
This order transparency is generally quite sensitive. I am not sure if there is a better way to do this but we should check with Kitware.

**Comment by Garyfallidis** [comment]
Also some cards do not support it. Maybe there is an update with vtk 8.2.0?

**Comment by skoudoro** [comment]
fixed by #87. closing
